ttcn3/types/types.go

```diff
@@ -146,7 +146,14 @@ type PrimitiveType struct {
 }
 
 func (t *PrimitiveType) String() string {
-	return ""
+	var constraints string
+	if t.ValueConstraints != nil {
+		for _, v := range t.ValueConstraints {
+			constraints = constraints + v.String() + ", "
+		}
+		constraints = " (" + strings.Trim(constraints, ", ") + ")"
+	}
+	return t.Kind.String() + constraints
```

Comment:
Consider using as slice for contraints:
```suggestion

	if t.ValueConstraints == nil {
		return t.Kind.String()
	}
	var constrains []string
	for _, v := range t.ValueConstraints {
		constraints = append(constraints, v.String())
	}
	return fmt.Sprintf("%s (%s)", t.Kind.String(), strings.Join(constraints, ", "))
```